tool/src/java/org/sakaiproject/lessonbuildertool/resolver/MultiCommonsMultipartResolver.java

```diff
@@ -0,0 +1,94 @@
+package org.sakaiproject.lessonbuildertool.resolver;
+
+import org.apache.commons.fileupload.FileItem;
+import org.springframework.http.MediaType;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.util.StringUtils;
+import org.springframework.web.multipart.MultipartException;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.multipart.commons.CommonsFileUploadSupport;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+import org.springframework.web.multipart.commons.CommonsMultipartResolver;
+
+import javax.servlet.ServletContext;
+import java.io.UnsupportedEncodingException;
+import java.nio.charset.Charset;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by neelam on 8/3/2015.
+ * This class extends CommonsMultipartResolver's parseFileItems() method to upload multiple files coming from same form input name.
+ * Linked to JIRA WL-4002
```

Comment:
Don't put WL-refs in the comments as we will send patches like this back to Sakai and they won't mean anything there.